cfgtool

```diff
@@ -100,11 +100,11 @@ vpn_pki_setup() {
 
   # Diffie Hellman parameters; this is a bunch of prime numbers used for crypto
   # stuff. DH key exchange is useful so a new key is computed for every connection.
-  # Even if our Server .key is stolen, attackers won't be able to decrypt past 
+  # Even if our Server .key is stolen, attackers won't be able to decrypt past
   # connections.
   # This takes a long time but greatly enhances security. It should be repeated
   # every couple of moths or so.
-  # There is a standard set of 1024bit DH parameters, but attacks are known 
+  # There is a standard set of 1024bit DH parameters, but attacks are known
   # against that. We are using 4096bit custom parameters, so that should be
   # much safer.
   evp easyrsa --batch gen-dh
```

Comment:
We may add a "no-regen-dh" switch or something here. Generating the DH parameters takes a long time and should not be necessary often